### ISSUE TITLE
 "could not create val_spec" is not very helpful

### ISSUE THREAD

**Issue by UnixJunkie** [mooreryan/ocaml_python_bindgen#6]
When one is missing the penultimate unit parameter in the *_specs.txt file,
this is the error message to get.
There could be a more helpful message, or at least that would be nice.

**Comment by UnixJunkie** [comment]
luckily, the function spec for which the generation failed is printed just before, but still
it took me some time to realize what was wrong with my spec.

**Comment by mooreryan** [comment]
Yeah, you've hit one of the items on my to do list...the error messages are generally pretty unhelpful.  I still haven't gotten the hang of getting angstrom to produce nice error messages, to be honest.  But I agree, they aren't that helpful as is, and need some work.

**Comment by mooreryan** [comment]
I added a bit more info to some of the failures.  I wouldn't call the problem fixed, but it's moving in a better direction at least.
